english/marshmallow.md

```diff
@@ -1040,3 +1040,4 @@ You wake up confused, you faint you go back in [time](time-travel/time-travel.md
 
 The password is [xyzzy](plugh/plugh.md)
 
+Perishing in an avalanche of words, in a mundane collection of sentences called "stories". [Devastated?](devastated_greatness/devastated_greatness.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.